src/slab.js

```diff
@@ -58,7 +58,8 @@ async function startInstanceRequest() {
 }
 
 async function waitForInstance(taskId, taskName) {
-  for (let i = 0; i < 60; i++) {
+  // while (true) equivalent to please ESLint
+  for (;;) {
```

Comment:
the action can be struc forever ?

Reply:
No because the task will finish at some point. Plus in case of task fetching failure we also break.

Reply:
plus worst case it's an ubuntu instance from github that is going to spin forever